SMCalloutView.m

```diff
@@ -186,23 +190,43 @@ - (CGFloat)innerContentMarginLeft {
 - (CGFloat)innerContentMarginRight {
     if (self.rightAccessoryView)
         return self.rightAccessoryHorizontalMargin + self.rightAccessoryView.$width + TITLE_HMARGIN;
+    else if(self.useCustomContentViewMargin)
+        return self.contentViewMargin.right;
     else
         return TITLE_HMARGIN;
 }
 
 - (CGFloat)calloutHeight {
-    return self.calloutContainerHeight + ANCHOR_HEIGHT;
+    if(self.backgroundView.anchorHeight)
+    {
```

Comment:
OK now I'm getting super picky but could you match the existing braces style (in this case no braces for a single statement)